vassal-app/src/main/java/VASSAL/tools/swing/SwingUtils.java

```diff
@@ -62,7 +65,7 @@ public static AffineTransform descaleTransform(AffineTransform t) {
     );
   }
 
-  public static final Map<?, ?> FONT_HINTS = (Map<?, ?>) Toolkit.getDefaultToolkit().getDesktopProperty("awt.font.desktophints");
+  public static final Map<?, ?> FONT_HINTS = GraphicsEnvironment.isHeadless() ? new HashMap<String, String>() : (Map<?, ?>) Toolkit.getDefaultToolkit().getDesktopProperty("awt.font.desktophints");
```

Comment:
I'd use Collections.emptyMap() instead of creating a mutable HashMap here for the headless case.

Also, is this necessary? Toolkit.getDefaultToolkit() returns a special headless one, so this shouldn't throw when run in headless mode.

Reply:
getDefaultTookit() is not the problem, The headleass toolkit returns null when you call  getDesktopProperty("awt.font.desktophints") which blows up the new ImagePicker I wrote on Travis when it initialises in a test.

Reply:
Oh, huh. I suppose it makes sense that this property isn't defined in headless mode. Good catch.